solvis/solution/inversion_solution/inversion_solution_file.py

```diff
@@ -260,6 +261,7 @@ def ruptures(self) -> 'DataFrame[RuptureSchema]':
         return cast('DataFrame[RuptureSchema]', df)
 
     @property
+    @lru_cache
```

Comment:
consider using `functools.cache` new in py3.9 (which is min python for `solvis`) https://docs.python.org/3/library/functools.html#functools.cache

Reply:
Agree .cache is better that lru_cache in this case (with no args) . I'll tweak that